src/test/java/codeu/model/data/UserTest.java

```diff
@@ -27,12 +28,13 @@ public void testCreate() {
     String name = "test_username";
     Instant creation = Instant.now();
     String password = "password";
+    String hashedPassword = BCrypt.hashpw(password, BCrypt.gensalt());
```

Comment:
This is actually not necessary to do in the test. You had a choice to make: put the password hashing in the User constructor (i.e., this.hashedPassword = BCrypt.hashpw(password, BCrypt.gensalt()))
which would leave all of the call sites unaffected and be a single change; or keep the User constructor the same and make the encryption at the call sites.  There are pros and cons to each.  If you centralize it all, it's pretty clean, and there's no chance that you encrpyt the password wrong.  If you don't, then you reserve the future right to change how you do encryption, on a per-user basis (not sure why you would, just saying you could).  So with the direction you chose, I think it doesn't matter if you've actually hashed the password or not, because User does not interpret it in any way.